ddtrace/contrib/internal/selenium/patch.py

```diff
@@ -0,0 +1,133 @@
+import os
+import time
+import typing as t
+
+import selenium
+from selenium.webdriver.remote.webdriver import WebDriver
```

Comment:
Can we avoid force-importing this and doing something similar to https://github.com/DataDog/dd-trace-py/blob/23ab0e6610ee8a4a1bbd0507ade14011651ea66f/ddtrace/contrib/internal/django/patch.py#L834 instead, that is, register a "when imported" hook to do the patching when that module is actually imported by the target application.

Reply:
Will do.